modules/account-lib/src/coin/baseCoin/iface.ts

```diff
@@ -130,3 +130,24 @@ export type FeeOptions = {
   formula?: 'fixed' | 'feeRate' | 'perKB' | 'custom';
   type?: 'base' | 'max' | 'tip';
 };
+
+/**
+ * The validity window signals when a transaction can be included in the network.
+ *  ValidityWindow information is stored as part of the transaction data in the indexer and wallet platform, unless it is absolutely necessary.
+ */
+export interface ValidityWindow {
+  firstValid?: number;
+  lastValid?: number;
+  minDuration?: number;
+  maxDuration?: number;
+  unit?: 'block' | 'seconds' | 'milliseconds';
+}
+
+/**
+ * Sequence Ids are used to differentiate payments coming from the same account and to prevent accidental double spends and replay attacks.
+ */
+export interface SequenceId {
+  name: string; // "Nonce", "Sequence Id", "Counter"
+  keyword: string; // "nonce", "sequenceId", "counter"
```

Comment:
should these be more specific?
e.g.
`name: 'Nonce' | 'Sequence Id' | ...`
`keyword: 'nonce' | 'sequenceId' | ....`